packages/ai-bot/lib/matrix.ts

```diff
@@ -132,6 +122,54 @@ export async function sendError(
   }
 }
 
+export const toMatrixMessageContent = (
+  functionCall: FunctionToolCall,
+  eventToUpdate: string | undefined,
+): IContent | undefined => {
+  let { arguments: payload } = functionCall;
+  if (functionCall.name === 'patchCard') {
+    const id = payload['card_id'];
+    const body = payload['description'] || "Here's the change:";
+    let messageObject: IContent = {
+      body: body,
+      msgtype: 'org.boxel.command',
+      formatted_body: body,
+      format: 'org.matrix.custom.html',
+      data: {
+        command: {
+          type: functionCall.name,
+          id: id,
+          patch: {
+            attributes: payload['attributes'],
+            relationships: payload['relationships'],
+          },
+          eventId: eventToUpdate,
+          toolCall: functionCall,
+        },
+      },
+    };
+    return messageObject;
+  } else if (functionCall.name === 'searchCard') {
+    const body = payload['description'] || "Here's the query:";
```

Comment:
This is another reason why the type for `arguments` is not `any`. You have specific assumptions around this property in the `arguments` for search--let's enforce via the type

Reply:
As in the previous PR, We will keep all this code in the command service for now until the command field api is introduced